package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "elastic-composer",
-    "version": "4.4.0",
+    "version": "4.6.0",
```

Comment:
Should be 4.5.0

Reply:
branch `master` is at ```    "version": "4.5.0", ``` so it will fail CI if it's not > than that

Reply:
Gotcha! In that case, maybe you should rebase?